src/routes.tsx

```diff
@@ -56,111 +56,116 @@ export const routes: RouteObject[] = [
         loader: searchParamLoader,
       },
       {
-        path: '/:network/:layer',
+        path: '/:network/*',
         element: <NetworkSpecificPart />,
-        errorElement: withDefaultTheme(<RoutingErrorPage />),
-        loader: scopeLoader,
         children: [
           {
-            path: '',
-            element: <ParatimeDashboardPage />,
-          },
-          {
-            path: 'search', // Search within this scope
-            element: <SearchResultsPage />,
-            loader: searchParamLoader,
-          },
-
-          {
-            path: `block`,
-            element: <BlocksPage />,
-          },
-          {
-            path: `block/:blockHeight`,
-            element: <BlockDetailPage />,
-            loader: blockHeightParamLoader,
-          },
-          {
-            path: `address/:address`,
-            element: <AccountDetailsPage />,
-            loader: addressParamLoader(),
+            path: ':layer',
```

Comment:
I'm not sure we need splat to add `path: 'consensus'` later
https://lukaw3d.github.io/react-router-v6-searchable-docs/start/overview/#ranked-route-matching 

Another issue will be `useParams`. It won't return layer=consensus in that case.
One idea:
```diff
  path: ':layer',
+ loader: async ({ params }) => params,
+ id: 'scope1',

  path: 'consensus',
  loader: async ({ params }) => ({...params, layer: consensus}),
  id: 'scope2',

- useParams
+ useLoaderData('scope1') ?? useLoaderData('scope2')
```